### PR TITLE
Plloyd/logging

### PR DESCRIPTION
**Pull request by PeteLloyd**

### CODE REVIEW

File: src/upload-course-content.js [hunk 1]
```diff
@@ -94,12 +95,17 @@ module.exports = class UploadCourseContent {
 
 	async _createModule(instanceUrl, orgUnit, module, parentModule) {
 		const url = parentModule
-			? new URL(`/d2l/api/le/1.34/${orgUnit.Id}/content/modules/${parentModule.Id}/structure/`, instanceUrl)
-			: new URL(`/d2l/api/le/1.34/${orgUnit.Id}/content/root/`, instanceUrl);
+			? new URL(`/d2l/api/le/1.34/${orgUnit.Identifier}/content/modules/${parentModule.ModuleId}/structure/`, instanceUrl)
```
Comment (khorwood) [inline src/upload-course-content.js:98]: I think `parentModule.Id` was correct:
https://docs.valence.desire2learn.com/res/content.html#Content.ContentObject

File: src/upload-course-content.js [hunk 2]
```diff
@@ -213,17 +226,9 @@ module.exports = class UploadCourseContent {
...
 
-		await this._fetch(
-			signedUrl,
-			{
-				method: 'PUT',
-				body: JSON.stringify(body)
-			});
-
-		const fileUrl = new URL(`/d2l/api/le/1.34/${orgUnit.Id}/content/topics/${lmsTopic.Id}/file`, instanceUrl);
+		const fileUrl = new URL(`/d2l/api/le/1.34/${orgUnit.Identifier}/content/topics/${lmsTopic.Identifier}/file`, instanceUrl);
```
Comment (khorwood) [inline src/upload-course-content.js:229]: I think `lmsTopic.Id` was correct:
https://docs.valence.desire2learn.com/res/content.html#Content.ContentObject

File: src/upload-course-content.js (hunk 1 above)
Comment (PeteLloyd) [reply src/upload-course-content.js:98]: Ah - I was looking at the TOC. Thanks for catching that - will fix.

**Comment by PeteLloyd** [comment]
wow, I botched that change :(   Should be fixed now.

**Pull request merged**